proteus/LinearAlgebraTools.py

```diff
@@ -131,6 +131,70 @@ def save(self, filename):
         _petsc_view(self, filename)
 
 
+class ParVec_petsc4pyTH(p4pyPETSc.Vec):
```

Comment:
@cekees - Hmm, I'm not sure this has anything to do with the BDM patch honestly.  I'm guessing the TH appended to the back refers to Taylor Hood.  Do you recall near the beginning of the year messing around with the parallel support for TH elements?  I think this had something to do with that.